packages/advisor-components/src/RuleDetails/RuleDetails.tsx

```diff
@@ -0,0 +1,247 @@
+import './RuleDetails.scss';
+
+import React, { MouseEventHandler, useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import Markdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import rehypeSanitize from 'rehype-sanitize';
+
+import { Split, SplitItem, Stack, StackItem, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+
+import InsightsLabel from '@redhat-cloud-services/frontend-components/InsightsLabel';
+import { SeverityLine } from '@redhat-cloud-services/frontend-components-charts/SeverityLine';
+
+import { IMPACT_LABEL_KEY, LIKELIHOOD_LABEL_KEY, RISK_OF_CHANGE_LABEL_KEY, TOTAL_RISK_LABEL_KEY } from '../constants';
+import RebootRequired from '../RebootRequired/RebootRequired';
+import RuleRating from '../RuleRating/RuleRating';
+import ContextWrapper, { ContextWrapperProps, ProductContext } from '../ContextWrapper';
+import messages from '../messages';
+import { barDividedList, topicLinks } from '../common';
+import { RiskOfChange } from '../types';
+import { Likelihood } from '../types';
+import { TopicRhel } from '../types';
+import { Rating } from '../types';
+import { RuleContentOcp } from '../types';
+import { RuleContentRhel } from '../types';
+import { AdvisorProduct } from '../types';
```

Comment:
These should be merged together
```suggestion
import { RiskOfChange, AdvisorProduct, Likelihood, TopicRhel, Rating, RuleContentOcp, RuleContentRhel } from '../types';
```